.github/workflows/python-test.yml

```diff
@@ -6,6 +6,11 @@ jobs:
 
   format:
```

Comment:
this is semantically incorrect. yet, i can't come up with something suited.

Reply:
"runchecks" will probably do.